elementary/monitor/dbt_project/macros/alerts/manage/update_sent_alerts.sql

```diff
@@ -2,11 +2,15 @@
     {% if execute %}
         {% if alert_ids %}
             {% set update_sent_alerts_query %}
-                UPDATE {{ ref('elementary_cli', 'alerts_v2') }} set status = 'sent', sent_at = {{ elementary.edr_quote(sent_at) }}, updated_at = {{ elementary.edr_current_timestamp() }}
-                WHERE alert_id IN {{ elementary.strings_list_to_tuple(alert_ids) }} and status = 'pending' and
-                    {{ elementary.edr_cast_as_timestamp('detected_at') }} >= {{ elementary_cli.get_alerts_time_limit() }}
+                update {{ ref('elementary_cli', 'alerts_v2') }}
+                set status = 'sent',
+                    sent_at = {{ elementary.edr_cast_as_timestamp(elementary.edr_quote(sent_at)) }},
```

Comment:
Hey, thank you for raising that up!
The reason `sent_at` was set to `edr_quote` and not a timestamp is because the old tables that we were using were setting it to `edr_quote` and not timestamp.

I have no problem with this change as it makes sense, but we need to make sure that we don't break things here for other adapters / backwards compatibility / users that already has sent_at as `edr_quote`

Reply:
We had a similar situation in the [following PR](https://github.com/elementary-data/elementary/pull/1318/files).
You can use the same implementation here - if relation already exists, check the type of the `sent_at` column and use it (could be string or timestamp). If not, use timestamp 🙂

Reply:
I'll take a look there. However as I see from ddl, field sent_at is of `timestamp` data type:
```
{% macro empty_alerts_v2() %}
    {{ elementary.empty_table([
		('alert_id', 'long_string'),
		('alert_class_id', 'long_string'),
		('type', 'string'),
		('detected_at','timestamp'),
    	('created_at','timestamp'),
    	('updated_at','timestamp'),
		('status','string'),
		('data', 'long_string'),
		('sent_at', 'timestamp')
    ]) }}
{% endmacro %}
```

Reply:
Missed that!
So it might be all good to implement that change without the backwards support.
Let me check that and get back to you 🙂

Reply:
Hi there @IDoneShaveIt ! Just kind reminder: can we move further with this PR ?

Reply:
Hey @svdimchenko, sorry for the late response (I was away).
I checked it and it seems like your solution will work (as we already using the column as timestamp as you mentioned) 🙂 

Approving the PR, thank you again for your contribution!